pkg/visitors/insert.go

```diff
@@ -0,0 +1,171 @@
+// Copyright 2018 the LinuxBoot Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package visitors
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/linuxboot/fiano/pkg/uefi"
+)
+
+// InsertType defines the insert type operation that is requested
+type InsertType int
+
+// Insert Types
+const (
+	// These first three specify a firmware volume.
+
+	// InsertFront inserts a file at the beginning of the firmware volume,
+	// which is specified by 1) FVname GUID, or  (File GUID/File name) of a file
+	// inside that FV.
+	InsertFront InsertType = iota
+	// InsertEnd inserts a file at the end of the specified firmware volume.
+	InsertEnd
+	// InsertIn inserts a file anywhere in the specified firmware volume.
+	// It works exactly like InsertEnd for now.
+	InsertIn
```

Comment:
Should we still have `InsertIn` if it's not really implemented yet? People might depend on this behaviour.

Reply:
What exactly would they depend on? that it inserts at the end? I could remove the comment and say that it's currently implemented like insertEnd but no one should depend on it